packages/admin-e2e-tests/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is this file needed? I think just using the main `tsconfig.json` might be preferable to keep all packages on the same config. @samueljseay what do you think?

Reply:
I think this is ok @adrianduffell every package will have its own basic settings that varies slightly from the others and it opens up the option to disable or enable some rules if we have to on a per package basis. What I'm glad to see here is that it extends the base config. :+1: